Example2_SDMMaxent.Rmd

```diff
@@ -25,27 +25,43 @@ knitr::opts_chunk$set(cache = 2, cache.rebuild = FALSE)
+When `SpaDES` modules are open and shared, this also expands a potential pool of users who can themselves help improve the code.
 
-## **Module creation and coding**
+## Module creation and coding
 
-With the above in mind, in this example we created three modules that source and pre-process data ('data modules') and a module that fits an SDM and iteratively forecasts species distributions (we call it a 'simulation module', although the simulation only involves predicting from a statistical model). The idea is that we could, for instance, provide different data sources to one of the data modules and only update the parts of the simulation that are affected by this change (i.e. presumably the other data module steps with not be affected). Or, we could develop a second simulation module using a different SDM approach and swap the two modules to inspect which provides better predictions.
+With the above in mind, in this example we created three modules that source and pre-process data ('data modules') and a module that fits an SDM and iteratively forecasts species distributions (we call it a 'simulation module', although the simulation only involves predicting from a statistical model).
+The idea is that we could, for instance, provide different data sources to one of the data modules and only update the parts of the simulation that are affected by this change (i.e. presumably the other data module steps with not be affected).
+Or, we could develop a second simulation module using a different SDM approach and swap the two modules to inspect which provides better predictions.
 
-Our data modules are *speciesAbundanceData* and *climateData*. The simulation module is *projectSpeciesDist*. We start by creating an `.R` script to set up and control the simulation. In this example this script is called `Example2_SDMMaxent.R`. The script begins with a few lines of code that ensure necessary packages are installed and loaded (see [Reproducible package installation]). It then defines the necessary folder directories for the simulation and creates the modules in the `modules/` folder:
+Our data modules are *speciesAbundanceData* and *climateData*.
+The simulation module is *projectSpeciesDist*.
+We start by creating an `.R` script to set up and control the simulation.
+In this example this script is called `Example2_SDMMaxent.R`.
+The script begins with a few lines of code that ensure necessary packages are installed and loaded (see [Reproducible package installation]).
+It then defines the necessary folder directories for the simulation and creates the modules in the `modules/` folder:
 
 ```{r globalscript, eval = FALSE}
 ## Get necessary R packages, but don't load any
 if (!require("Require")) install.packages("Require")
 Require::Require("PredictiveEcology/SpaDES.install@development")
 SpaDES.install::installSpaDES(ask = TRUE)
+Require("PredictiveEcology/SpaDES.core@development")
```

Comment:
Please don't load packages yet. It messes up with `SpaDES.install::makeSureAllPackagesInstalled`
SpaDES is loaded after `SpaDES.install::makeSureAllPackagesInstalled` right before setting up the simulation

Reply:
yup, i should have done `require = FALSE`, but i removed this line